Core/Session/GrindingSession.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Core.Session
+{
+    public class GrindingSession : IGrindingSession
+    {
+        private readonly IBotController _botController;
```

Comment:
Personally I don't like the naming, using the `_` as a prefix. in the constructor you can use the `this.` keyword instead.

Reply:
i'm getting used to underscore naming conventions for private method etc.
do you have fx cop or something like that i can change my coding style to fit to the repo.

Reply:
well there's no coding convention for the project which set in stone yet. Personally I've just tried to use the already set coding style.